src/pages/portfolio/index.tsx

```diff
@@ -1,11 +1,49 @@
 'use client';
 
+import React from 'react';
+import { XCircle } from 'lucide-react';
+import { Button } from '@penumbra-zone/ui/Button';
 import { Text } from '@penumbra-zone/ui/Text';
+import { Density } from '@penumbra-zone/ui/Density';
+import mobile from 'is-mobile';
 
 export const PortfolioPage = () => {
-  return (
-    <section>
-      <Text h2>Hi!</Text>
+  const isOnMobile = mobile();
+
+  return isOnMobile ? (
+    <section className='absolute inset-0 flex flex-col items-center justify-between p-4 gap-3 border-t  border-neutral-800'>
+      <div className='flex flex-col justify-center items-center p-0 gap-4 w-full flex-grow'>
+        <div className='relative'>
+          <XCircle className='text-neutral-400  w-8 h-8' />
```

Comment:
suggestion (namings): use here and everywhere else the named color classes, e.g. `text-neutral-light` instead of `text-neutral-400`. This might actually be easier knowing that Figma shows the named colors next to its usage:

<img width="903" alt="Screenshot 2024-11-21 at 10 33 33" src="https://github.com/user-attachments/assets/733c84d3-b116-4c11-aa18-8f35b91c0829">

Reply:
nitpick: those double spaces in classNames make my eyes twitch 😂

Reply:
sorry, will fix :D